scootapi/setup/setup.go

```diff
@@ -1,3 +1,6 @@
+// Package setup provides Cloud Scoot Setup, which is a mechanism for
```

Comment:
This is the only thing I didn't directly address, wasn't sure how to go about this.